### ISSUE TITLE
how does the loss converge

### ISSUE THREAD

**Issue by jyshee** [happynear/AMSoftmax#25]
Does the AMS loss have the similar converge curve with softmax loss? In my exps, the AMS loss (set m=1) changes little during training, even after lots of iters.

**Comment by happynear** [comment]
Loss is meaningless in AMS. Maybe you can output the accuracy, target logit etc to observe the convergence of the network.
